RBI-5: Create header of the main page

FarcasDavid opened the pull request:
Added the text that says “Hello, NAME” and “What would you like to cook today?”

Also, added the search button on the right side but with no functionality.

Commit:
RBI-5:
-Added the welcome and title labels on the home screen
-Added search button on the right side

edwardchristian97 commented on RecipeBook/ViewControllers/HomeScreenViewController/HomeScreenViewController.swift:
```diff
@@ -0,0 +1,24 @@
+import UIKit
+
+class HomeScreenViewController: UIViewController {
+
```
Remove whitespace

edwardchristian97 commented on RecipeBook/ViewControllers/HomeScreenViewController/HomeScreenViewController.swift:
```diff
@@ -0,0 +1,24 @@
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+
```
Remove whitespace

edwardchristian97 commented on RecipeBook/ViewControllers/NameScreenViewController/NameScreenViewController.swift:
```diff
@@ -14,9 +14,26 @@ class NameScreenViewController: UIViewController {
     @IBOutlet private weak var nameTextField: UITextField!
 
     override func viewDidLoad() {
+
```
Remove whitespace

edwardchristian97 commented on RecipeBook/ViewControllers/NameScreenViewController/NameScreenViewController.swift:
```diff
@@ -14,9 +14,26 @@ class NameScreenViewController: UIViewController {
+extension NameScreenViewController: UITextFieldDelegate {
+
+    func textFieldShouldReturn(_ textField: UITextField) -> Bool {
+        print("Return pressed")
```
Unnecessary print, remove it

Commit:
RBI-5:
- Updated Search Button Position, Size and Aspect in Home Screen
- Code refactoring
